### ISSUE TITLE
Run ORNL OpenSHMEM Benchmarks

### ISSUE THREAD

**Issue by jdinan** [Sandia-OpenSHMEM/SOS#661]
Run OSB in testing: https://github.com/ornl-languages/osb/releases

**Comment by wrrobin** [comment]
@jdinan, is our plan to add OSB in Travis tests? In such case, should we run all the apps in all envs (e.g. single and multi-threaded)? Or a selected set?

**Comment by jdinan** [comment]
Depends on how long they run and whether they improve testing coverage over existing tests.  Could you review the tests and make a recommendation?  For example, we might want to treat these like other performance benchmarks and track performance over time.

**Comment by wrrobin** [comment]
Sure. I will check and do some test runs.

**Comment by wrrobin** [comment]
Here are my thoughts. All the tests that I have done is using np = 4, keeping similar to Travis.
1. Graph 500: Useful application to check since we dont have anything yet in our tests. However, takes too long. So, adding it in Travis would be a disaster. 
2. SSCA1: Not sure whether the benchmark has an issue or it is just too slow. Seems to be not an ideal option for Travis.
3. GUPs: We already have our own gups implementation in tests. Not sure whether the minor differences need to be tested. 
4. NAS: It has only IS which we are already getting from ISx.
5. SHOMS: Useful benchmarking for all the APIs (128 total) but takes a long time to run. We can configure the msg size to make it shorter for Travis. The number of iterations seems to be derived and cannot be configured. 

In summary, none of these tests is suitable for Travis. But, we can definitely add them in Jenkins so that we check the tests from time to time. Specially, Graph500 and SHOMS. Please let me know your suggestions.

**Comment by jdinan** [comment]
Thanks for looking into this.  Adding to Jenkins sounds good, but also sounds like we can put this at lower priority.

**Comment by jdinan** [comment]
Discussed running in Jenkins and determined:
1. Graph 500 -- Ok, would be good to run in testing
2. SSCA1 -- Skip
3. GUPS -- Skip, already have our own version
4. NAS -- Skip, already have ISX
5. SHOMS -- ???, good to run if it works reliably
